observability/logging/logging_5/log-visualization.adoc

```diff
@@ -0,0 +1,15 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+[id="log-visualization"]
+= About log visualization
+:context: log-visualization
+
+toc::[]
+
+You can visualize your log data in the {product-title} web console.The following section describes configure the legacy log visualizer based on Kibana. The instructions are meant for the use-cases where ElasticSearch is still used. 
```

Comment:
```suggestion
You can visualize your log data in the {product-title} web console. The following section describes how to configure the legacy log visualizer based on Kibana. The instructions are meant for the use-cases where ElasticSearch is still used. 
```

Reply:
Elasticsearch support is EOF on Openshift 4.15 for Logging. The elasticsearch are kept for trace. I don't think we need to backport it into Logging 6.2